packages/api/test/mutations.test.ts

```diff
@@ -0,0 +1,54 @@
+import type { GraphQLSchema } from 'graphql'
+import { execute, parse } from 'graphql'
+
+import { getSchema, getContextFactory } from '../src'
+import {
+  ValidateCartMutation,
+  InvalidCart,
+  ValidCart,
+} from '../mocks/ValidateCartMutation'
+
+let schema: GraphQLSchema
+let context: Record<string, any>
+
+beforeAll(async () => {
+  schema = await getSchema({
+    platform: 'vtex',
+    account: 'storecomponents',
+    environment: 'vtexcommercestable',
+    channel: '1',
+  })
+
+  const contextFactory = getContextFactory({
+    platform: 'vtex',
+    account: 'storeframework',
+    environment: 'vtexcommercestable',
+    channel: '1',
+  })
+
+  context = contextFactory({})
+})
+
+test('`validateCart` mutation, returning `null`', async () => {
```

Comment:
What do you think about:
```suggestion
test('`validateCart` mutation should return `null` when a valid cart is passed', async () => {
```